2016/2016.md

```diff
@@ -0,0 +1,21 @@
+## 2016
+
+| Conference | Date | Venue | Description |
+|------------|------|-------|-------------|
+|[Mini Deb-Con](http://in2015.mini.debconf.org/)|30 - 31 January| Andheri (E), Mumbai | It is an awesome opportunity for any user who wants to experience the magic of FOSS |
```

Comment:
The list order is no more chronological now. It's reverse chronological (see https://github.com/nikhita/tech-conferences-india/pull/29).

Can you update the list to how it occurs in the current README now?